src/fides/api/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -1633,6 +1633,7 @@ def create_privacy_request_func(
             privacy_request_data.requested_at,
             policy.id,
             config_proxy.execution.subject_identity_verification_required,
+            authenticated,
```

Comment:
When privacy requests are created from the workflow of updating consent preferences, they are automatically treated as authenticated as they would have been authenticated via the consent request workflow.  Passing in that authenticated value here lets us put the privacy request in the correct state.